addons/payment_stripe/controllers/main.py

```diff
@@ -15,6 +15,8 @@ class StripeController(http.Controller):
     def stripe_s2s_create_json(self, **kwargs):
         acquirer_id = int(kwargs.get('acquirer_id'))
         acquirer = request.env['payment.acquirer'].browse(acquirer_id)
+        if not kwargs.get('partner_id'):
```

Comment:
As you have 6 times the same code couldn't we extract it somewhere else to make it unique and less likely to be broken and/or forgotten when making changes ?

Reply:
I up that one.

Reply:
I saw, but I don't know how to do it, as there is no super-function to this, and the call is made from different places too, so the problem would be the same. The last option I see is to create a new function that add the partner in the global payment controller, but I find this is a little overkill, I'm just adding the user to the params...